### ISSUE TITLE
HTTP ports exposed in interact hook do not match the dockerfile EXPOSE 

### ISSUE THREAD

**Issue by cmacdonald** [osirrc/jig#112]
Hence its difficult to document which ports a user should connect to.
See for example this from Terrier:
![image](https://user-images.githubusercontent.com/620938/60343501-08a58280-99ac-11e9-820d-db1c662114ad.png)

The jig _could_ simply assume that it should expose the ports as they were requested.

**Comment by arjenpdevries** [comment]
I think (...) that the first time you run a docker exposing that port, it actually uses the port you request - but then the port is not available for the next time you run a docker exposing that port, so it automatically maps to a different one.

**Comment by r-clancy** [comment]
Yeah, by default Docker uses a random port on the host side so that containers that require the same port don't collide (i.e., we could both run a MySQL container on a server without colliding with each other).

We can add in some explicit port mappings within the `jig` (after the workshop, currently have no bandwidth) that are passed through to Docker.

**Comment by arjenpdevries** [comment]
After starting jig in interact mode, this gives you the actual mapping: `docker port $(docker ps -aql)`
